example/C/src/ContainerizedFederatedHelloWorld/HelloWorld.lf

```diff
@@ -0,0 +1,67 @@
+/**
+ * Containerized distributed LF program where a MessageGenerator creates a string
```

Comment:
I suggest it might be better to put this in `example/C/src/DistributedHelloWorld/` but in a separate folder, called `docker` for example. This way, you can utilize the [import](https://github.com/lf-lang/lingua-franca/wiki/Language-Specification#import-statement) system in LF to import from `HelloWorld.lf` and avoid duplicate code.